### DISCUSSION TITLE
Network error / RemoteProtocolError

### DISCUSSION THREAD

**Discussion by beckstring** [acheong08/ChatGPT#340] [Q&A] [answered]
Hi, 

First of all, thanks for designing this repo, highly appreciated!

Do you guys have any workarounds for the RemoteProtocolError that you receive when the answer from ChatGPT requires more than 60 seconds?

When regularly using the website this error appears as "Network error". It is super annoying and highly limits the usability of the model compared to the last week...

**Comment by acheong08** [comment, marked as answer]
If you use the streaming option, you will get the response up until the error. Unlike the website, it won't delete the previously streamed response.

There is no way for me/us to fix their servers though

**Reply by acheong08** [reply]
Btw this is no longer true. The network error has been fixed on their side
